### PR TITLE
add simonbyrne to Julia maintainer list

### PR DESCRIPTION
**Pull request by simonbyrne**
cc: @tkelman @staticfloat @ninjin

### CODE REVIEW

**Comment by tkelman** [comment]
also L34

**Comment by simonbyrne** [comment]
done.

File: lib/travis/build/script/julia.rb [hunk 1]
```diff
@@ -30,8 +31,8 @@ def setup
             ansi: :green
           sh.echo '  https://github.com/travis-ci/travis-ci/issues' \
             '/new?labels=julia', ansi: :green
-          sh.echo 'and mention \`@tkelman\`, \`@ninjin\` and ' \
-            '\`@staticfloat\` in the issue', ansi: :green
+          sh.echo 'and mention \`@tkelman\`, \`@ninjin\`, \`@staticfloat\`' \
```
Comment (tkelman) [inline lib/travis/build/script/julia.rb]: space before and, or after staticfloat?

Comment (simonbyrne) [inline lib/travis/build/script/julia.rb]: ah, fixed now.

**Pull request merged**